biggerquery/cli.py

```diff
@@ -130,35 +134,64 @@ def read_project_name_from_setup() -> Optional[str]:
 
 def build_project_name_description(project_name: str) -> str:
     if project_name is None:
-        return 'Project name not found in the project_setup.PROJECT_NAME.'
+        return ''
     else:
         return 'Project name is taken from project_setup.PROJECT_NAME: {0}.'.format(project_name)
 
 
-def find_root_package(project_name: Optional[str], root: Optional[str]) -> Path:
+def find_root_package(project_name: Optional[str], project_dir: Optional[str]) -> Path:
```

Comment:
If you make something optional, then provide default as None

Reply:
it's not mine code, I've only renamed a parameter